### ISSUE TITLE
[FEATURE] twitchnotify 2 dm

### ISSUE THREAD

**Issue by zekroTJA** [zekroTJA/shinpuru#215]
The twitchnotify module should also be able to send notifications via DM, if desired.

**Comment by shiipou** [comment]
I hope that wont be DM to all member of a guild.
Because of spam, I didn't want to became spam of notif in DM just because of an admin want me to receive a notif.

**Comment by zekroTJA** [comment]
> I hope that wont be DM to all member of a guild. Because of spam, I didn't want to became spam of notif in DM just because of an admin want me to receive a notif.

Nope, I am generally against such methods. The only thing where multiple people will receive DMs unasked is when an antiraid alert is triggered. Then, all admins of a guild will get a DM, if enabled.

This system would be an opt-in solution to "subscribe" to DM notifications for twitch notifications. This would of course also be out-out-able the same way.
